canvas_sdk/v1/data/task.py

```diff
@@ -99,10 +98,45 @@ class Meta:
     task_label = models.ForeignKey(TaskLabel, on_delete=models.DO_NOTHING, null=True)
     task = models.ForeignKey(Task, on_delete=models.DO_NOTHING, null=True)
 
+class NoteTask(Model):
+    """Note Task."""
+
+    class Meta:
+        db_table = "canvas_sdk_data_api_notetask_001"
+
+    note = models.ForeignKey("v1.Note", on_delete=models.DO_NOTHING, related_name="note_tasks", null=True)
+    task = models.ForeignKey(Task, on_delete=models.DO_NOTHING, related_name="note_tasks", null=True)
+    patient = models.ForeignKey(
+        "v1.Patient",
+        on_delete=models.DO_NOTHING,
+        null=True,
+    )
+    original_title = models.TextField()
```

Comment:
can you add `blank=True, default=""` to this one? we're trying to get these properties to match up to what's in home-app